README.md

```diff
@@ -21,10 +21,13 @@ This package follows semver.
   - `/connectors/withPageProductId` - Provides decoded `productId` read from a pathname of a product page. [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withPageProductId)
   - `/connectors/withPageState` - Provides `isVisible` and `isLoading` booleans. Values are taken from a page state. [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withPageState)
   - `/connectors/withUser` - Provides `user` object with user data [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withUser)
```

Comment:
withThemeComponents + withProductContext mention is missing here

Reply:
done

Reply:
done